packages/atomic/src/components/search-box-suggestions/atomic-search-box-query-suggestions/atomic-search-box-query-suggestions.tsx

```diff
@@ -45,6 +51,11 @@ export class AtomicSearchBoxQuerySuggestions {
     }
   }
 
+  get Icon() {
+    if (this.icon === '') return null;
+    return this.icon || SearchIcon;
```

Comment:
I prefer `??` when we want to fallback from `null` or` undefined`.

```suggestion
    return this.icon ?? SearchIcon;
```